Automattic/jetpack
# Stats: Changed dashboard widget form input name to allow settings to save

Fixes #20575

#### Changes proposed in this Pull Request:

* The previous `stats_jetpack_dashboard_widget` form input name was `widget_id`, which was causing some issues (not being correctly picked up as a POST variable). Changing the form input name (changed to `stats_id`) enables the settings in the dashboard widget to be saved, when changed.

#### Other information:


#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Visit the main wp-admin dashboard (yoursite.com/wp-admin/), and scroll down to the Jetpack Stats widget
* Click on the gear icon (top right), and change any of the drop-down options.

![Screenshot 2022-07-08 at 14 45](https://user-images.githubusercontent.com/16754605/178005315-2fc241bd-d932-42e2-89cf-7eb98d452b02.png)


* Click on submit and the page should refresh
* Without this PR, the option should go back to it's default value.
* Apply this PR, and check that the settings have been correctly saved (and that the stats chart or post / search information reflects the change)
